order-api/src/main/java/com/zerobase/gamesell/order/config/JwtConfig.java

```diff
@@ -0,0 +1,21 @@
+package com.zerobase.gamesell.order.config;
+
+import com.user.gamedomain.config.JwtAuthenticationProvider;
+import com.user.gamedomain.util.Aes256Util;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class JwtConfig {
+
+  private final Aes256Util aes256Util;
+
+  public JwtConfig(Aes256Util aes256Util) {
+    this.aes256Util = aes256Util;
+  }
+
+  @Bean
+  public JwtAuthenticationProvider jwtAuthenticationProvider() {
```

Comment:
Aes256Util가 bean이므로 param으로 주입받는 것이 좀더 깔끔할 것 같네요
```suggestion
  public JwtAuthenticationProvider jwtAuthenticationProvider(Aes256Util aes256Util) {
```